### PR TITLE
Doc improvements

### PR DESCRIPTION
**Pull request by bonsanto-stripe**
Fixed typos in some texts, corrected translations and added few steps

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -83,17 +83,17 @@
           <h1>¡Gracias! Solo falta un paso más:</h1>
           <div class="sr-instructions">
             <p>
-              Instrucciones de pago en OXXO:
-              <p>1. Entregue el código de al cajero en cualquier OXXO para que lo escanee.</p>
-              <p>2. Proporcione el pago en efectivo al cajero.</p>
+              Instrucciones para realizar el pago en OXXO:
```
Comment (fay-stripe) [inline client/index.html:86]: Let's make sure the text here is consistent with the text in the integration guide: https://site-admin.stripe.com/docs/payments/oxxo#oxxo-payment-instructions. If we get these translations approved, let's change the integration guide to match

File: server/README.md [hunk 2]
```diff
@@ -6,6 +6,6 @@ Pick the language you are most comfortable in and follow the instructions in the
 # Supported languages
 
 * [JavaScript (Node)](node/README.md)
+* [PHP (Slim)](php/README.md)
 * [Python (Flask)](python/README.md)
-* [Ruby (Sinatra)](ruby/README.md)
```
Comment (fay-stripe) [inline server/README.md:10]: Is this for alphabetical order? I like it!

File: README.md [hunk 3]
```diff
@@ -55,7 +55,8 @@ git clone https://github.com/stripe-samples/oxxo-payment
 Copy the .env.example file into a file named .env in the folder of the server you want to use. For example:
 
 ```
-cp .env.example server/node/.env
+cd oxxo-payment
+cp .env.example server/<node|php|python|ruby>/.env
```
Comment (fay-stripe) [inline README.md:61]: If you want to include all the options here instead of just an example, let's remove "For example:" on line 55

**Review by fay-stripe — CHANGES_REQUESTED** [review]
Thanks for making these OXXO Stripe Sample improvements! I left a few comments, and here's one additional thing to note: 
Because the text has been updated, we'll want to update the GIF that shows the demo: https://github.com/stripe-samples/oxxo-payment#demo

File: server/README.md (hunk 2 above)
Comment (bonsanto-stripe) [reply server/README.md:10]: Yup, that was the intention

File: client/index.html (hunk 1 above)
Comment (fay-stripe) [reply client/index.html:86]: Looks like we've incorporated the translation updates from https://docs.google.com/document/d/1gsvYhRS1csIVzsg3p1YjA-zWS5nV6Fa4G5M1BO_jf-4/edit?disco=AAAAGevGr9w&ts=5eb09271&usp_dm=true
Thanks!

Comment (bonsanto-stripe) [reply client/index.html:86]: All suggestions made by belen were made

**Review by fay-stripe — APPROVED** [review]
🎉

**Comment by fay-stripe** [comment]
Nice! 🚢

**Pull request merged**